src/core/pretty.ml

```diff
@@ -205,6 +208,15 @@ let pp_command : p_command pp = fun oc cmd ->
       List.iter (out " %a" pp_p_arg) args;
       Option.iter (out " :@ @[<hov>%a@]" pp_p_term) ao;
       out " ≔@ @[<hov>%a@]@]" pp_p_term t
+  | P_inductive(e,s,t,tl)   ->
+      out "@[<hov 2>%ainductive %a" pp_expo e pp_ident s;
+      Option.iter (out " :@ @[<hov>%a@]" pp_p_term) t;
+      List.iter (pp_pair pp_ident pp_p_term oc (" ≔@ @[<hov>%a@] :@ @[<hov>%a@]")) tl
```

Comment:
":=" should be printed before List.iter. And List.iter should print "|" before each case, except perhaps the first one.